deployment.template.json

```diff
@@ -148,6 +162,9 @@
               "image": "${MODULES.Grafana}",
               "createOptions": {
                 "HostConfig": {
+                  "Binds": [
+                    "$REDIS_UNIX_SOCKET_VOL:$REDIS_UNIX_SOCKET_DIR"
```

Comment:
This binding needs to be done on every container that is accessing Redis?

Reply:
Correct, all Redis clients (our C# module, redis-exporter and Grafana) are accessing the socket file `redis.sock` instead of the 6379 TCP port.